tests/platform_tests/thermal_control_test_helper.py

```diff
@@ -280,7 +280,44 @@ def restart_thermal_control_daemon(dut):
     config_reload(dut)
     assert 0, 'Wait thermal control daemon restart failed'
 
+def stop_thermal_control_daemon(dut):
+    """
+    Stop thermal control daemon
+    """
+    logging.info('Stopping thermal control daemon on {}...'.format(dut.hostname))
+    find_thermalctld_pid_cmd = 'docker exec -i pmon bash -c \'pgrep -f thermalctld\' | sort'
+    output = dut.shell(find_thermalctld_pid_cmd)
+    assert output["rc"] == 0, "Run command '%s' failed" % find_thermalctld_pid_cmd
+    assert len(output["stdout_lines"]) >= 2, "There should be at least 2 thermalctld process"
+                                                                    
+    stop_thermalctl_cmd = "docker exec -i pmon bash -c 'supervisorctl stop thermalctld'"
+    output = dut.shell(stop_thermalctl_cmd)
+    if output["rc"] == 0:
+        assert len(output["stdout_lines"]) >= 1, "There should be at least 1 thermalctld process"
```

Comment:
Can we use 'supervisorctl status' command to check the status instead of checking the output line numbers of the previous commands?

Reply:
thanks Sujin! will update these 4 places